target/classes/META-INF/resources/home_page.html

```diff
@@ -40,6 +40,9 @@
                         <li class="nav-item dropdown">
                             <a class="nav-link" href="\item_pages\general.html">General Items</a>
                         </li>
+                        <li class="nav-item-dropdown">
+                            <a class="nav-link" href="\donation_page\donation_form.html">Donation Form</a>
+                        </li>
                     </ul>
                 </div>
             </div>
```

Comment:
Files under the `target` directory are generated when the server is run and shouldn't be in the GitHub repository. The directory is listed in the `.gitignore`, so they shouldn't have been tracked, but somehow they were added awhile ago and I didn't notice. It's fixed in the main branch now, so if you branch from main in the future the files shouldn't get added to your commits, but for now can you remove this file from the pull request so it doesn't get added to the tracked files again?
Removing the file with `git rm --cached target/classes/META-INF/resources/home_page.html' or just deleting it should work

Reply:
Okay I removed it using the command you commented